src/pages/guides/using/marketo/marketo-lead-activity-data-stream-setup.md

```diff
@@ -300,70 +305,84 @@ And finally, all Attribute objects in the `attributes` array will contain the fo
 | dataType  | String | The data type of the attribute |
 | value     | Any    | The value of the attribute     |
 
+You are always able to emit a sample event from within the Adobe Developer Console to see the exact structure of the event.
```

Comment:
I'm fine to have this note here, but let's change it from exact structure to see a sample structure for that specific event type, as the sample could still differ from the exact events they receive, as there can be a lot of variance.